ethcore/src/client/client.rs

```diff
@@ -1121,21 +1147,22 @@ impl MiningBlockChainClient for Client {
 	}
 
 	fn import_sealed_block(&self, block: SealedBlock) -> ImportResult {
-		let _import_lock = self.import_lock.lock();
-		let _timer = PerfTimer::new("import_sealed_block");
-		let start = precise_time_ns();
-
 		let h = block.header().hash();
-		let number = block.header().number();
+		let start = precise_time_ns();
```

Comment:
this all done due to additional locking or alteration of lock semantics? would be nice to have a comment for what's going on.